components/datetime/benches/datetime.rs

```diff
@@ -170,6 +170,21 @@ fn datetime_benches(c: &mut Criterion) {
         });
     });
 
+    group.bench_function("fixed/format_to_string/ymd_numeric/10_cases", |b| {
```

Comment:
please add to_string benches for the other fieldsets (at least a non numeric one), i think to_string is the more important thing to bench here, and the difference may be relevant for numeric vs non numeric because of length

Reply:
My view is that most of these benches should be to_string, and we can have mirrored write ones for some or all of them, but covering to_string is more important than covering write.